src/cow-react/constants/routes.ts

```diff
@@ -1,9 +1,11 @@
+export const LIMIT_ORDERS_PATH = 'limit-orders'
+
 // ENUM with routes
 export enum Routes {
   HOME = '/',
   SWAP = '/swap',
   SWAP_OUTPUT_CURRENCY = '/swap/:outputCurrency',
-  LIMIT_ORDER = '/limit-order',
+  LIMIT_ORDER = '/:chainId?/limit-orders/:inputCurrencyId?/:outputCurrencyId?',
```

Comment:
I think I mentioned this in another PR: can you use the const from line 1 here?

Reply:
It's not possible:
<img width="861" alt="image" src="https://user-images.githubusercontent.com/7122625/196652318-c3905367-2eb9-4ae1-a40c-4e421bd6de4c.png">